enrollment-server-onboarding-provider-innovatrics/src/main/java/com/wultra/app/onboardingserver/provider/innovatrics/InnovatricsPresenceCheckProvider.java

```diff
@@ -164,12 +164,7 @@ private static String fetchCustomerId(final OwnerId id, final SessionInfo sessio
     public void cleanupIdentityData(final OwnerId id, final SessionInfo sessionInfo) throws PresenceCheckException, RemoteCommunicationException {
         logger.info("Invoked cleanupIdentityData, {}", id);
         final String customerId = fetchCustomerId(id, sessionInfo);
-
-        innovatricsApiService.deleteLiveness(customerId, id);
-        logger.debug("Deleted liveness, {}", id);
-
-        innovatricsApiService.deleteSelfie(customerId, id);
-        logger.debug("Deleted selfie, {}", id);
+        innovatricsApiService.deleteCustomer(customerId, id);
```

Comment:
According to the documentation:
> Deleting a customer will remove any related data, such as selfies and document pages. Otherwise, the data will expire after a configured amount of time.